src/fetch_tse_data.py

```diff
@@ -0,0 +1,225 @@
+"""
+This script downloads and format some data from TSE website.
+The first objective with this data is to obtain a list of all politicians in Brazil.
+In march 2017, the data available in TSE website contained information about elected people from the year 1994 to 2016.
+Data before 1994 does not contains name of the politicians.
+Further, they inform that data from 1994 to 2002 is insconsistent and they are working on it.
+The data is available in csv format: one csv file per state, grouped in one zip file per year.
+Some of the csv files from TSE contain headers.
+Unfortunately, this is not the case for the files we are dealing with here.
+For different years there are different numbers of columns, and consequently, different headers.
+In this script, after downloading the files, we appropriately name the columns and select a useful subsample of columns to export for future use in Serenata Project.
+"""
+
+import pandas as pd
+import numpy as np
+import os
+import urllib
+import zipfile
+import glob
+
+from tempfile import mkdtemp
+TEMP_PATH= mkdtemp()
+
+FILENAME_PREFIX= 'consulta_cand_'
+TSE_CANDIDATES_URL= 'http://agencia.tse.jus.br/estatistica/sead/odsele/consulta_cand/'
+OUTPUT_DATASET_PATH= os.path.join(os.pardir,'data','2017-03-31-tse-candidates.xz')
```

Comment:
It's not a good idea to have this date hardcoded here. Something like this might be useful:

```console
>>> from datetime import date
>>> today = date.today()
>>> today.strftime('%Y-%m-%d')
'2017-05-10'
```